lib/suppressions.cpp

```diff
@@ -445,3 +446,22 @@ const std::list<Suppressions::Suppression> &Suppressions::getSuppressions() cons
 {
     return mSuppressions;
 }
+
+/**
+ * This function require that both tokens and suppressions are sorted in line
```

Comment:
That assumption will not work when files are included.

Reply:
OK? You mean that both suppression and token list can be multiple files in some 'list'??

Reply:
As far as I know the suppressions are listed in arbitrary order. If you want to sort them feel free to do it.

The `tokenizer.tokens()` contains all tokens in a translation unit.

If you have a file `foo.h` with 100 lines of code.. then you put this code in foo.c:

    #include "foo.h"
    int x;

then the `tokenizer.tokens()` will first contain the tokens from "foo.h" and then the "int x;" tokens.. so if you just look at the line number it will decrease here.